jupyter-extension/jupyterlab_pachyderm/mount_server_client.py

```diff
@@ -37,8 +46,10 @@ def __init__(
             self.sock_path = ""
         self.client = AsyncHTTPClient()
         # non-prived container flag (set via -e NONPRIV_CONTAINER=1)
-        # TODO: Would be preferable to auto-detect this, but unclear how
+        # or use DET_RESOURCES_TYPE environment variable to auto-detect this.
         self.nopriv = NONPRIV_CONTAINER
+        if DET_RESOURCES_TYPE == SLURM_JOB:
+            self.nopriv = 1
```

Comment:
I think adding a logging statement here that NONPRIV is being inferred would be helpful.

Reply:
> I think adding a logging statement here that NONPRIV is being inferred would be helpful.

ok